jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/coder/stream/LogFileToAsn1CodecPublisher.java

```diff
@@ -170,53 +162,30 @@ private void publishList(XmlUtils xmlUtils, List<OdeData> dataList) throws JsonP
 				publisher.publish(JsonUtils.toJson(odeData, false),
 					publisher.getOdeProperties().getKafkaTopicOdeRawEncodedSPATJson());
 			} else {
-				// Determine the message type (MAP or TIM are the current other options)
-				String messageType = determineMessageType(msgPayload);
+				// Determine the message type (MAP, TIM, SSM, SRM, or PSM)
+				String messageType = UperUtil.determineMessageType(msgPayload);
```

Comment:
Not necessarily in this PR but as a backlog item, what about switching this UperUtil to return an enum rather than string?